python/llm/example/GPU/StableDiffusion/README.md

```diff
@@ -0,0 +1,124 @@
+# Stable Diffusion
+In this directory, you will find examples on how to run StableDiffusion models on [Intel GPUs](../README.md).
+
+### 1. Installation
+#### 1.1 Installation on Linux
+We suggest using conda to manage environment. 
+```bash
+conda create -n diffusion python=3.11
```

Comment:
remove the conda env creation part but use `conda activate diffusion` instead, as ipex-llm installation part has created it.